File: src/model/TypeOfNoticeEnum.ts
export enum TypeOfNoticeEnum {
    INFORMATION = 'Informação',
    WARNING = 'Alerta',
    IMPORTANT = 'Importante'
}

export const TypeOfNoticeList: {
    key: string;
    value: string;
}[] = Object.entries(TypeOfNoticeEnum).map(([key, value]) => ({ key, value }))

export type TypeOfNoticeType = keyof typeof TypeOfNoticeEnum

export const TypeOfNoticeEntries = Object.entries(TypeOfNoticeEnum)

File: src/main/java/br/com/api/enums/TypeOfNoticeEnum.java
package br.com.api.enums;

import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
@Getter
public enum TypeOfNoticeEnum {

	INFORMATION("label.information"), 
	WARNING("label.warning"), 
	IMPORTANT("label.important");

	private final String i18n;
}
